util/reggen/countermeasure.py

```diff
@@ -0,0 +1,68 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, List
+
+from .lib import check_keys, check_str, check_list
+
+# TO BE DEFINED
+CM_ASSETS = [
+    'FOO'
+]
+
+# TO BE DEFINED
+CM_TYPES = [
+    'BAR'
+]
+
+
+class CounterMeasure:
+    def __init__(self, asset: str, cmtype: str, desc: str):
+        self.asset = asset
+        self.cmtype = cmtype
+        self.desc = desc
+
+    @staticmethod
+    def from_raw(what: str, raw: object) -> 'CounterMeasure':
+        rd = check_keys(raw, what, ['name', 'desc'], [])
+
+        name = check_str(rd['name'], 'name field of ' + what)
+        desc = check_str(rd['desc'], 'desc field of ' + what)
+
+        try:
+            asset, cmtype = name.split('.')
+        except ValueError:
+            raise ValueError('The countermeasure ID {} ({}) has the '
+                             'wrong format.'
+                             .format(name, what))
```

Comment:
nit: 
```python
            raise ValueError(f"Invalid countermeasure format: {name} ({what}).")
```
similar below, for brevity.

Reply:
done

Reply:
Yeah let me shorten all of these error messages.